psutil/tests/test_process.py

```diff
@@ -1426,6 +1427,7 @@ def clean_dict(d):
 
     @unittest.skipIf(not HAS_ENVIRON, "not supported")
     @unittest.skipIf(not POSIX, "POSIX only")
+    @unittest.skipIf(MACOS_11PLUS, "macOS 11+ not supported, issue #2084")
```

Comment:
> This is because environ() for a process != getpid() rerturns an empty dict, correct?

Yes, I'll try to reword the message. I meant `test_weird_environ` is not supported on macOS 11+ rather than `environ()`.